a/arione-joy.md

```diff
@@ -0,0 +1,27 @@
+# Arione Joy
+
+![arione](arione-joy.jpg)
```

Comment:
This is the right filename, but it'll break, because the actual image doesn't have a matching filename.